packages/components/src/modal/style.scss

```diff
@@ -98,7 +98,7 @@
 	flex-direction: row;
 	justify-content: space-between;
 	align-items: center;
-	height: $header-height + $grid-unit-15;
+	height: $header-height + $grid-unit-10;
```

Comment:
Required to maintain the current modal header height which is 72px.

Reply:
Maybe another good spot for adding a comment?